.env.example

```diff
@@ -33,6 +33,7 @@ WCRS_EMAIL_SERVICE_EMAIL=""
 AWS_DAILY_EXPORT_ACCESS_KEY_ID=
 AWS_DAILY_EXPORT_SECRET_ACCESS_KEY=
 AWS_DAILY_EXPORT_BUCKET=
+AWS_BOXI_EXPORT_BUCKET=
```

Comment:
Only comment is that these need to come in 3's

- Access Key ID
- Secret Access Key
- Boxi export bucket

We save a bit of time and money by using the same credentials and buckets in non-production environments, but often they are different when it comes to production.

So we need the flexibility to specify a different access key ID and secret access key for BOXI, as well as the bucket name.

Reply:
WIll add them in this PR :)